app/assets/stylesheets/govuk-component/_previous-and-next-navigation.scss

```diff
@@ -1,81 +1,83 @@
-.govuk-previous-and-next-navigation {
+.pub-c-pagination {
   display: block;
   margin-top: $gutter;
   margin-bottom: $gutter;
   margin-left: -$gutter-half;
   margin-right: -$gutter-half;
```

Comment:
It's less about them always being the same and more about having to understand how the function expects something to be passed.

With long hand you know exactly how it'll behave without having to consult the docs if you forget.

If you put the two versions in front of developers, there will be a lot of people who couldnt tell you what the short hand is doing whereas everyone will be able to tell you what the long hand does.

I like explicit code that doesnt require someone from having to consult documentation.